src/Maker/MakeUser.php

```diff
@@ -98,6 +102,15 @@ class_exists(DoctrineBundle::class)
             if ($missingPackagesMessage) {
                 throw new RuntimeCommandException($missingPackagesMessage);
             }
+
+            $question = 'Enter the entity\'s table name. (<comment>It will be auto-escaped.</comment>)';
+            $entityTableName = $io->ask(
+                $question, 'user', function ($name) {
+                return Validator::validateDoctrineTableName($name, $this->doctrineHelper->getRegistry());
+            });
+
+            $command->addOption('entity-table-name', $entityTableName);
+            $input->setOption('entity-table-name', $entityTableName);
```

Comment:
I'd prefer not to ask a new question to fix an edge-case bug. In most cases, I don't want the user to think about their table name. Suggestion:

1) Move this logic to `EntityClassGenerator::generateEntityClass()`

2) Inside that method, use the `doctrine.orm.configuration` service to fetch the naming strategy and check of the *potential* table name is invalid. Here's fake code:

```php
        /** @var NamingStrategy $namingStrategy */
        $namingStrategy = $this->doctrineConfiguration
            ->getNamingStrategy();
        $entityClassDetails->getShortName();

        $potentialTableName = $namingStrategy
            ->classToTableName($entityClassDetails->getFullName());
        if ($this->connection->getDatabasePlatform()->getReservedKeywordsList()->isKeyword($potentialTableName)) {
            // add ORM\Table
        }
```

Reply:
I didn't perceived this as an edge case: a lot of projects use prefixes for their tables, so this feature prevent the need for the user to add the annotation by itself. 
From my point of view, this adds more granularity for the maker.

Reply:
Of course, if you still think it's not a good idea, i'll amend the code with your suggestion ;)

Reply:
Yea, I think we should remove the question and just add the `@ORM\Table` automatically *when* needed. The topic whether we should ask this question everywhere that we create entities is a separate thing. And really, if people are prefixing all their database tables (which sometimes makes sense - I have done that before), we should make that easier to do on a global level in Doctrine (so that you don't need the `@ORM\Table` everywhere).

Reply:
i'm working on that, but i'm facing a problem: `doctrine.orm.configuration` is abstract, thus i need to inject `doctrine.orm.default_configuration`. that means this feature will only work on default connection.

And if none exists (which *can* happen, i think) ==> :boom:

Reply:
hmm actually `ManagerRegistry` does have a method `getManager()`, but it only returns a `ObjectManager` and not an `EntityManagerInterface`, so we don't have access to method `getConfiguration()`

Reply:
Yep, let's cheat. But check with an `instanceof` and if it's not an `EntityManager`, let's throw an exception.

Reply:
ok it's done... but now, aren't we lacking a test on postgreSQL for instance?

Reply:
That's very true. I think we should live with that for now.